internal/venice-client-common/src/main/java/com/linkedin/venice/controllerapi/D2ServiceDiscoveryResponse.java

```diff
@@ -4,19 +4,28 @@
 
 
 public class D2ServiceDiscoveryResponse extends ControllerResponse {
-  String d2Service;
+  String routerD2Service;
```

Comment:
We cannot change the `d2Service` field since this information gets serialized and deserialized as Json and there are clients that already rely on the `d2Service` fields. It is okay to add a new `serverD2Service` field